LEAF_Request_Portal/sources/Group.php

```diff
@@ -388,37 +362,29 @@ public function importUser(string $userID, int $groupID, string $backupID): arra
      */
     public function deactivateMember($member, $groupID): void
```

Comment:
Can we add type hinting here or will it break things?

Reply:
I'm not sure if it will break anything or not. I did add the PHPDoc section so we know what should be passed to this method.